.github/workflows/release.yml

```diff
@@ -102,3 +103,10 @@ jobs:
           files: dist/science-*
           fail_on_unmatched_files: true
           discussion_category_name: Announcements
+      - name: Add Binaries to ${{ needs.determine-tag.outputs.release-tag }} Release
```

Comment:
The Windows machines were messing up the release notes; so I added this fork. We only need release notes and discussion creation to be triggered once since they are not unique per platform like the binaries are; so this makes sense anyhow.